client/utils.py

```diff
@@ -0,0 +1,121 @@
+"""
+Contains some common functions of the related app
+to store and retrieve data from database.
+"""
+from django.contrib.auth.models import User
+from django.shortcuts import get_object_or_404
+from openedx.features.openedx_nodebb_discussion.models import EdxNodeBBUser, EdxNodeBBCategory
+
+
+def save_user_relation_into_db(username, nodebb_uid):
+    """
+    Saves NodeBB uid against edx_userid in EdxNodeBBUser table.
+
+    Args:
+        username (str): edX username of user
+        nodebb_uid (int): NodeBB uid for edX user
+    """
+    relation = EdxNodeBBUser()
+    relation.edx_uid = get_object_or_404(User, username=username)
+    relation.nodebb_uid = nodebb_uid
+    relation.save()
+
+
+def get_nodebb_uid_from_username(username):
+    """
+    Extract nodebb_uid from table EdxNodeBBUser using username.
+
+    Args:
+        username (str): edX username of user
+
+    Returns:
+        int: returns nodebb_uid get from model
+    """
+    edx_user = get_object_or_404(User, username=username)
+    user_relation = get_object_or_404(EdxNodeBBUser, edx_uid=edx_user)
```

Comment:
Can't we use single query for this purpose ?

Reply:
We are saving our data against an edx_user object in the model so first, we have to get edx_user from User table and then for extracting nodebb_uid we match the user in the model and get uid.